api_guard/dist/types/ajax/index.d.ts

```diff
@@ -33,12 +33,14 @@ export interface AjaxError extends Error {
 
 export declare const AjaxError: AjaxErrorCtor;
 
+export declare type AjaxHttpMethod = 'GET' | 'HEAD' | 'POST' | 'PUT' | 'DELETE' | 'CONNECT' | 'OPTIONS' | 'TRACE' | 'PATCH' | '';
```

Comment:
```suggestion
/**
 * A valid HTTP method as defined in the registry in RFC-7231 by the IETF, Section 8.1.3
 * https://tools.ietf.org/html/rfc7231#section-8.1.3
 */
export declare type HttpMethod = 'GET' | 'HEAD' | 'POST' | 'PUT' | 'DELETE' | 'CONNECT' | 'OPTIONS' | 'TRACE' | 'PATCH';
```

I think we probably shouldn't allow `''` here. Also I wanted to add a comment that points to the specification so people know we're not just pulling these out of our butts.

Reply:
You are right, i added [here](https://github.com/ReactiveX/rxjs/pull/6237/files/886a69c674ef783a682bf3a2d7553622ecb46435#r616369336)